tools/ci-build.sh

```diff
@@ -248,17 +249,16 @@ fi
 if [ -e build ]; then /bin/rm -rf build; fi
 mkdir -p ${P4C_DIR}/build
 cd ${P4C_DIR}/build
-cmake ${CMAKE_FLAGS} ..
+cmake ${CMAKE_FLAGS} -G Ninja ..
 
 # If CMAKE_ONLY is active, only run CMake. Do not build.
 if [ "$CMAKE_ONLY" == "OFF" ]; then
-  make -j$((`nproc`+1))
-  sudo make -j$((`nproc`+1)) install
+  cmake --build . -- -j $(nproc)
```

Comment:
Note that you do not need `-j` with ninja. It defaults to all cpu cores available.

Reply:
In case you use make or something else as generator we may want to preserve the parallelism.